src/main/java/lottoauto/wrapper/Lotto.java

```diff
@@ -0,0 +1,66 @@
+package lottoauto.wrapper;
+
+import lottoauto.util.InputNumberValidator;
+import lottoauto.util.RandomNumberExtractor;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private List<Integer> numbers;
```

Comment:
```suggestion
    private List<Number> numbers;
```

1~45까지의 숫자를 가지는 `Number` 클래스가 있으면 어떨까요?

그리고 `Number` 클래스가 직접 관련된 로직을 제어하게 된다면 도메인 별 책임이 좀더 분명하게 나뉠수 있을것 같아요.